test/sampleApp/index.html

```diff
@@ -510,6 +502,21 @@ <h3>Wrapped in a <code>&lt;figure&gt;</code> element, with a <code>&lt;figcaptio
                         
                     </p>
                 </fieldset>
+
+                <fieldset id="expectedTextSpec">
```

Comment:
shouldn't  selector in test be updated to expectedTextSpec too ?

Reply:
Its not in use.